lblod/frontend-gelinkt-notuleren
# Support snippet RDFa linking

### Overview
Just use the latest PR versions of plugins and editor to enable snippet RDFa linking.

Draft until tagged versions exist.

##### connected issues and PRs:
Plugins: https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/462
Editor: https://github.com/lblod/ember-rdfa-editor/pull/1212
RB frontend: https://github.com/lblod/frontend-reglementaire-bijlage/pull/263
RB backend: https://github.com/lblod/app-reglementaire-bijlage/pull/81
For Jira ticket [GN-4898](https://binnenland.atlassian.net/browse/GN-4898)

### Setup
Update `regulatoryStatementEndpoint` environment setting to point to local app-RB with PR version of resource config.
Set `@shouldEditRdfa={{true}}` on agendapoints edit page.

### How to test/reproduce
Follow steps 1 and 2 on [RB frontend PR](https://github.com/lblod/frontend-reglementaire-bijlage/pull/263) to create a template with linked snippet placeholder.
- Use that template to create an agendapoint
- Select a snippet placeholder and insert a snippet with imported resource links
- Save the document (this reloads the editor, which is for the moment necessary to view the links in the tools)
- Verify that the links are created correctly.

### Challenges/uncertainties
...

### Checks PR readiness
